### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cnovak**
Added more information explaining project better, highlighting contributions, and more details on support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
...
-for Drupal.
+This project automates the installation of [Apigee Kickstart developer portal](https://www.drupal.org/project/apigee_devportal_kickstart)
+for Drupal using [Ansible](https://docs.ansible.com/).
 
-This playbook is intended to be a starting point, and you may want to fork this project
-to make your own changes to the `playbook.xml` and other files for your own
-requirements.
-
-This playbook will install:
+By using this project, you will be able to quickly get Apigee Kickstart developer portal installed on a Linux system, including all dependencies. This Ansible playbook will install:
```
Comment (jbennett13) [inline README.md:5]: can

Comment (jbennett13) [inline README.md:5]: can

File: README.md [hunk 2]
```diff
@@ -60,7 +64,7 @@ You can add issues and ask for help by putting an issue into this repository's
 issue queue.
 
 ## Contributing
-Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
+This is an open source project and encourage users to contribute. Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
 
 ## Tests
 This project comes with a simple testing tool to validate the playbook installs
```
Comment (jbennett13) [inline README.md:70]: Add period.

File: README.md (hunk 1 above)
Comment (jbennett13) [inline README.md]: "you can"

File: README.md [hunk 3]
```diff
@@ -60,7 +64,7 @@ You can add issues and ask for help by putting an issue into this repository's
 issue queue.
 
 ## Contributing
-Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
+This is an open source project and encourage users to contribute. Read the [contributing guide](docs/contributing.md) to learn more about making contributions.
```
Comment (jbennett13) [inline README.md]: "and users are encouraged to contribute."

File: README.md (hunk 2 above)
Comment (jbennett13) [inline README.md]: Add a period to the end of this sentence?

**Review by jbennett13 — APPROVED** [review]
LGTM - a few tiny suggestions.  :)

File: README.md [hunk 4]
```diff
@@ -15,16 +11,24 @@ This playbook will install:
 - Drupal
 - Apigee Kickstart Drupal profile
 
-The topology can be one host with everything, or two hosts where one is the 
-database, and the other is the web server which contains the Drupal site.
+This playbook is intended to be a starting point and not a "one size fits all" playbook. We recommend you fork this project and make changes to the `playbook.xml` and other files to fit your specific needs.
+
+The topology can be one host with everything, or two hosts where one is the database, and the other is the web server which contains the Drupal site.
+
+## Supported Platforms
```
Comment (earth2marsh) [inline README.md]: ## Target Platforms
so as to not imply an expectation of support?

File: README.md (hunk 3 above)
Comment (earth2marsh) [inline README.md]: This is an open source project that encourages contributions from the community.

File: README.md [hunk 5]
```diff
@@ -71,7 +75,8 @@ tests:
 ## License
 Apache 2.0
 
-This is not an officially supported Google product.
+## Support
+This is not an officially supported Google product. If you find a bug, have questions or want to give feedback, add an issue to the [project's issue queue](https://github.com/apigee/devportal-kickstart-drupal-ansible-installer/issues). If you have issues running Ansible or with one of the open source [Ansible roles included](https://github.com/apigee/devportal-kickstart-drupal-ansible-installer/blob/master/requirements.yml) in this project, you can work with those projects to find solutions to your issue.
```
Comment (earth2marsh) [inline README.md]: This is not an officially supported Google product. The architectural choices of these scripts do not connote support implications. Please refer to the extensive [documentation on drupal.org](https://www.drupal.org/docs/user_guide/en/installation-chapter.html) for guidance on installation.

**Review by earth2marsh — COMMENTED** [review]
made some recommended changes

File: README.md (hunk 1 above)
Comment (cnovak) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (cnovak) [reply README.md]: I don't see the missing .

File: README.md (hunk 4 above)
Comment (cnovak) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (cnovak) [reply README.md]: I took @earth2marsh suggestion below, hope that works

Comment (cnovak) [reply README.md]: Took your change over @jbennett13

**Review by earth2marsh — APPROVED** [review]
LGTM

**Pull request merged**